### PR TITLE
refs[TASK][31158]:Create UI: Detail Team

### PR DESCRIPTION
**Pull request by Domhoan**
## Related Tickets
- [#Task31158: Create UI: Detail Team](https://edu-redmine.sun-asterisk.vn/issues/31158)
## WHAT
- Create UI: Detail Team
## HOW
- Create UI: Detail Team: AppBarLayout+CollapsingToolbarLayout for background, RecyclerView for Player Item
image

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/activity_detail_team.xml [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+        app:layout_behavior="@string/appbar_scrolling_view_behavior">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <androidx.constraintlayout.widget.Guideline
+                android:id="@+id/guideline2"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
```
Comment (NguyenTaTrung) [inline app/src/main/res/layout/activity_detail_team.xml]: tên id k có nghĩa

File: app/src/main/res/layout/activity_detail_team.xml [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+                android:layout_width="0dp"
+                android:layout_height="@dimen/dp_32"
+                android:background="@color/color_athens_gray"
+                android:gravity="center_vertical"
+                android:text="@string/title_coach"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+
```
Comment (NguyenTaTrung) [inline app/src/main/res/layout/activity_detail_team.xml]: thừa dòng

File: app/src/main/res/layout/activity_detail_team.xml [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+                app:layout_constraintBottom_toBottomOf="@+id/imageCoachTwo"
+                app:layout_constraintStart_toStartOf="@+id/textNameCoachTwo"
+                app:layout_constraintTop_toBottomOf="@+id/textNameCoachTwo" />
+
+            <include
+                android:id="@+id/includeGoalkeeper"
+                layout="@layout/title_info_player"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="8dp"
```
Comment (NguyenTaTrung) [inline app/src/main/res/layout/activity_detail_team.xml]: hard code nữa nè

File: app/src/main/res/layout/title_info_player.xml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:background="@color/color_athens_gray">
+
+    <ImageView
+        android:id="@+id/imageView4"
+        android:layout_width="@dimen/dp_16"
+        android:layout_height="@dimen/dp_16"
```
Comment (NguyenTaTrung) [inline app/src/main/res/layout/title_info_player.xml]: đặt tên có nghĩa chú mấy imageView dưới cũng vậy nhé

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -28,4 +28,8 @@
     <string name="goal_difference">%s/%s</string>
     <string name="title_statistic">Statistic</string>
     <string name="title_select_date">Select Date</string>
+    <string name="text_forwards">Forwarder</string>
+    <string name="text_defender">Defender</string>
+    <string name="text_midfielder">Midfielder</string>
+    <string name="title_goalkeeper">Goalkeeper</string>
```
Comment (NguyenTaTrung) [inline app/src/main/res/values/strings.xml]: text, title, msg, error,... tách riêng ra chú. Có thể sắp xếp theo bảng chữ cái thì càng tốt

File: app/src/main/res/layout/activity_detail_team.xml [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+            app:contentScrim="@color/color_green_malachite"
+            app:layout_scrollFlags="scroll|exitUntilCollapsed">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <ImageView
+                    android:id="@+id/backgroundDetailTeam"
+                    android:layout_width="match_parent"
```
Comment (huytq-2422) [inline app/src/main/res/layout/activity_detail_team.xml]:
```suggestion
                    android:layout_width="0dp"
```

**Pull request merged**